app/webpacker/controllers/producers_select_controller.js

```diff
@@ -0,0 +1,37 @@
+import TomSelect from "./tom_select_controller";
+
+const minimumOptionsForSearchField = 11;
+
+export default class extends TomSelect {
+  static values = { options: Object, placeholder: String };
+
+  connect(options = {}) {
+    const template = this.template();
+    let additional_options = {};
+    const lines_of_option = template.content.children.length; 
+
+    if (lines_of_option < minimumOptionsForSearchField) {
+      additional_options.plugins = [];  
+    }
+
+    super.connect(additional_options);
+    this.control.on('focus', this.fetchProducers.bind(this));
+  }
+
+  disconnect() {
+    if (this.control) this.control.destroy();
+  }
+  
+  fetchProducers() {
+    if ((Object.keys(this.control.options).length) > 1) return false;
```

Comment:
What if there is only one option? It will try to populate this on every focus, right?

Reply:
Yes, on each focus, fetchProducers will be called.

At first, there is only one option (the selected one), so the select will be populated.
After that, the number of options will always be superior to 1 and there will be not new attempt to populate the select.

Reply:
Actually, if the user only has access to one producer, it will always be 1. So we could add an additional check below, eg:
```
if(template.content.children.length <= 1) return; # by the way, is the false return value necessary?
```